.github/workflows/macos.yml

```diff
@@ -61,8 +61,7 @@ jobs:
         env:
           HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: "TRUE"
         run: |
-          source scripts/setup-macos.sh
-          brew install $MACOS_BUILD_DEPS $MACOS_VELOX_DEPS
+          scripts/setup-macos.sh
```

Comment:
The previous setup seems to want to only install the brew deps and use bundling for everything else.
Since we have two builds `macos-13, macos-14` we can may be make one BUNDLED and the other SYSTEM.

`folly_SOURCE: BUNDLED` below seems to be not required.